wmn-data.json

```diff
@@ -2266,6 +2266,18 @@
         "m_code" : 200,
         "known" : ["gettr", "support"],
         "cat" : "social",
+        "valid" : true
+       },        
+       {
+        "name" : "gfycat",
+        "uri_check" : "https://gfycat.com/@{account}",
+        "post_body" : "",
+        "e_code" : 200,
+        "e_string" : "(@",

```

Comment:
This `e_string` is too simple and runs the risk of having false positives. Suggest using `gfycat-username`

Reply:
@WebBreacher: `Done` re change of `e_string` for `gfycat`